packages/airnode-admin/jest.config.js

```diff
@@ -4,16 +4,12 @@ module.exports = {
   projects: [
     {
       ...config,
-      // Add custom settings below
-      name: 'e2e',
```

Comment:
Why is it ok to remove the `name` property? What was it good for?

Reply:
I am not sure (didn't find it even in docs for older version). I suspect that this property was removed at some point and now jest started to issue warnings on unnecessary properties.